src/App.js

```diff
@@ -12,10 +14,17 @@ function App() {
     <>
       <header className="App-header">
         <div className="logo-title">
+          <RiFlickrFill className="icon1" />
+          <RiFlickrLine className="icon2" />
+          <p className="signal gill">BELL</p>
+          <CiWifiOn className="icon3" />
           <img src={Logo} className="App-logo" alt="logo" />
-          <h1 className="title">MET TOP TEN ARTISTS</h1>
+          <h1 className="title gill">MET</h1>
         </div>
-        <NavBar />
+        <NavBar className="bar" />
+        <CiBluetooth className="bluetooth" />
+        <p className="level">100%</p>
+        <CiBatteryFull className="battery" />
       </header>
       <Routes>
         <Route path="/" element={<Artists />} />
```

Comment:
- [ ] Dear @JuanDBta, for the homepage, we need to have a Home page that has a list of categories that could be `filtered using given parameters`, e.g. category name. In your case, I can see you have a list of artists, but there is no way to filter this item based on any specific attribute. Please make sure there is a way to filter elements inside the homepage.

Reply:
- Well resolved 👍🏽